docs/spec/spec.md

```diff
@@ -29,37 +29,29 @@ This specification elaborates on the `Persist CLI Tool` commands.
 ## 2. Initializing the Bal Project with Persistence Layer
 
 ```bash
-bal persist init
+bal persist init --datastore="datastore" --module="module_name"
 ```
 
+| Command Parameter |                                       Description                                        | Mandatory | Default Value |
+|:-----------------:|:----------------------------------------------------------------------------------------:|:---------:|:-------------:|
+|    --datastore    |  used to indicate the preferred database client. Currently, only 'mysql' is supported.   |    No     |     mysql     |
+|     --module      |      used to indicate the persist enabled module in which the files are generated.       |    No     |               |
+
+
 The command initializes the bal project with the persistence layer. This command includes the following steps,
 
 1. Create persist directory
    This directory should contain all data model definition files. This file will define the required entities as per the [`persist` specification](https://github.com/ballerina-platform/module-ballerina-persist/blob/main/docs/spec/spec.md#2-data-model-definition)
 2. Create a model definition file in persist directory
-   It will create a file with same name as the package name if no files are present in the `persist` directory. It will create an empty file with required imports(`import ballerina/persist as _;`).
-3. Create database_configurations.bal file inside the `default` module in the generated directory if the definition filename inside the `persist` directory is the same as the Ballerina package name. If not, create the bal file inside the submodule named as the definition file name.
-   This file contains configuration variables to initialize the data stores
+   It will create a file with same name as the module environment variable if no files are present in the `persist` directory. It will create an empty file with required imports(`import ballerina/persist as _;`).
```

Comment:
```suggestion
   It will create a file named `model.bal` with required imports(`import ballerina/persist as _;`),  if no files are present in the `persist` directory.
```